server/src/main/scala/controllers/application/WithImports.scala

```diff
@@ -36,4 +56,127 @@ trait WithImports {
       Future(response)
     }
   }
+
+  def simulationImport(): Action[MultipartFormData[Files.TemporaryFile]] = authByRole(ArrivalSimulationUpload) {
```

Comment:
Add validation around all possible input errors to enable user friendly feedback to the user